src/lib_json/json_value.cpp

```diff
@@ -248,8 +266,14 @@ Value::CZString::CZString(CZString&& other)
 #endif
 
 Value::CZString::~CZString() {
-  if (cstr_ && storage_.policy_ == duplicate)
-    releaseStringValue(const_cast<char*>(cstr_));
+  if (cstr_ && storage_.policy_ == duplicate) {
+#if JSON_USE_SECURE_MEMORY
+	  releaseStringValue(const_cast<char*>(cstr_), storage_.length_ + 1u); //+1 for null terminating character for sake of completeness but not actually necessary
```

Comment:
The code is the same between these 2 macro cases. I guess the comment is only meant to point out that the length is not even used in the other case. But we could eventually remove the `ifdef`, right?